syntax = "proto3";

import "datagen.proto";

service SchedulerServer{
	rpc InferRequest(JsonResponse) returns (JsonResponse) {}
	rpc FinishedRequest(JsonResponse) returns (JsonResponse) {}
}
